src/vsphere_cpi/spec/unit/cloud/vsphere/resources/datastore_spec.rb

```diff
@@ -1,60 +1,125 @@
 require 'spec_helper'
+require 'rspec/its'
 
 describe VSphereCloud::Resources::Datastore do
-  subject(:datastore) {
-    VSphereCloud::Resources::Datastore.new('foo_lun', datastore_mob,  true, 16 * 1024, 8 * 1024)
-  }
-  subject(:datastore_inaccess) {
-    VSphereCloud::Resources::Datastore.new('foo_lun', datastore_mob,  false, 16 * 1024, 8 * 1024)
-  }
+  subject do
+    described_class.new('foo_lun', mob, true, 16 * 1024, 8 * 1024)
+  end
```

Comment:
please remove unnecessary blank lines and you can apply rubymine indentation settings per https://www.pivotaltracker.com/n/projects/1133984/stories/134478283.

Reply:
I did not find any blank spaces or line in this file. Did you intend to mention some other files?